### DISCUSSION TITLE
Numpad navigation keys

### DISCUSSION THREAD

**Discussion by yerlaser** [rust-windowing/winit#2964] [Q&A]
When NumLock is off the keys on Numpad supposedly send key events that are different from equivalent keys on the main keyboard.
Other terminals allow to distinguish, e.g., NumpadArrowLeft from ArrowLeft.
Is it possible to add to this library as well?

**Comment by kchibisov** [comment]
They keys on numpad are different to the keys on the regular board. They have a location `Numpad` attached to them.
